terraform/stacks/main/outputs.tf

```diff
@@ -783,12 +783,12 @@ output "csb" {
       }
     }
     rds = {
-      host     = module.csb_broker.rds_host
-      port     = module.csb_broker.rds_port
-      url      = module.csb_broker.rds_url
-      name     = module.csb_broker.rds_name
-      username = module.csb_broker.rds_username
-      password = module.csb_broker.rds_password
+      host     = module.csb_broker[0].rds_host
```

Comment:
How do I modify outputs.tf so it uses the `csb_broker_stig` code in the dev environment (instead of `csb_broker`)